genie-agent/src/main/resources/application.yml

```diff
@@ -3,6 +3,9 @@ cloud:
   aws:
     credentials:
       instanceProfile: true
+    region:
```

Comment:
This might produce the same end result in terms of stuff loaded, but is it also the same in terms of speed?
Just curious and wondering if you did this benchmark.

Reply:
I didn't do any benchmarking. I believe Spring will still be scanning the classpath for auto configs and running an if check whether the class in question is a member of the ignore list. Now it'll still find the class but it'll check the property via the existing conditional on the auto config class. So in theory I imagine it is similar. Also this provides the benefit that someone could turn it back on if they want.